src/Components/Aspire.Azure.Storage.Blobs/ConfigurationSchema.json

```diff
@@ -77,6 +77,29 @@
                           "format": "uri",
                           "description": "Gets or sets the secondary storage 'System.Uri' that can be read from for the storage account if the\naccount is enabled for RA-GRS.\n\nIf this property is set, the secondary Uri will be used for GET or HEAD requests during retries.\nIf the status of the response from the secondary Uri is a 404, then subsequent retries for\nthe request will not use the secondary Uri again, as this indicates that the resource\nmay not have propagated there yet. Otherwise, subsequent retries will alternate back and forth\nbetween primary and secondary Uri."
                         },
+                        "Request100ContinueOptions": {
```

Comment:
Where does this stuff come from?

Reply:
This is the tooling that @eerhardt wrote to make sure the configuration schema for the configuration options in components stay up-to-date. Essentially each time that one of our transitive dependencies adds properties to their options type (which we then use in Aspire), our build infra will detect it and fail the build saying that the configuration schema needs an update and it will also show the command you need to run in order to update it.

Reply:
All of this file is auto-generated. The description and info are grabbed  from the tripple slash comments extracted by our build tooling from those transitive dependencies.